src/controllers/attendanceManagement/getAttandanceOfASessionByDate.js

```diff
@@ -45,11 +45,12 @@ const getAttendanceOfASessionByDate = async (req, res) => {
             let recordObj = {
               'memberId':enrolments[i].memberId,
               'attended':false,
-              'comment':''
+              'comments':''
```

Comment:
what is the purpose of renaming this field?

Reply:
actually in modelsits comments not comment andinresponse it was not coming due to comment.